compactcert/worker_test.go

```diff
@@ -385,9 +385,17 @@ func TestLatestSigsFromThisNode(t *testing.T) {
 	// Wait for a compact cert to be formed, so we know the signer thread is caught up.
 	_ = <-s.txmsg
 
-	latestSigs, err := w.LatestSigsFromThisNode()
-	require.NoError(t, err)
-	require.Equal(t, len(latestSigs), len(keys))
+	var latestSigs map[basics.Address]basics.Round
+	var err error
+	for x := 0; x < 100; x++ {
```

Comment:
change the loop so that x = 0 .. 10, and update the sleep time so that it would sleep 2 ^ x millisecond.